certs_governance/src/contract.rs

```diff
@@ -140,23 +129,22 @@ impl GovernanceTrait for CertGovernance {
 
     /// Get all relevant contract data.
     fn info(e: Env) -> Info {
-        let expiration_time = match read_expiration_time(&e) {
-            Some(value) => OptU64::Some(value),
-            None => OptU64::None,
-        };
         Info {
             name: read_name(&e),
             revocable: read_revocable(&e),
-            expiration_time,
+            expiration_time: read_expiration_time(&e),
             distribution_limit: read_distribution_limit(&e),
             supply: read_supply(&e),
         }
     }
 }
 
-/// Calculates the expiration date of a distributed Chaincert (using Unix time).
-fn expiration_date(e: &Env, distribution_date: u64) -> Option<u64> {
-    read_expiration_time(e).map(|exp_time| distribution_date + exp_time)
+/// Calculates the expiration date of a distributed Chaincert (using Epoch Unix Timestamp).
```

Comment:
```suggestion
/// Calculates the expiration date of a distributed Chaincert (using Epoch Unix Timestamp and Epoch time).
```

I think it's important to clarify that the distribution_date must be an **Epoch Unix Timestamp** but also that the expiration_time must be an **Epoch time**, what do you think? 🤔